packages/airnode-examples/package.json

```diff
@@ -29,7 +29,7 @@
     "run-airnode-locally": "ts-node src/scripts/run-airnode-locally.ts",
     "sponsor-requester": "ts-node src/scripts/sponsor-requester.ts",
     "stop-local-airnode": "ts-node src/scripts/stop-local-airnode.ts",
-    "test": "jest --selectProjects unit",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest --selectProjects unit",
```

Comment:
This change is required because prettier v3 [made the migration to using ECMAScript Modules](https://prettier.io/blog/2023/07/05/3.0.0), which `jest` has only [experimental support for](https://jestjs.io/docs/ecmascript-modules). The fix is adapted from their option number 2.